src/components/templates/pageTemplate/index.tsx

```diff
@@ -54,12 +25,7 @@ const PageTemplate = ({ children, title }: Props) => {
             </Head>
             <div>
                 <header>
-                    <Header
-                        urlLogo={urlLogo}
-                        width="60px"
-                        height="60px"
-                        listNavLinks={listNavLinks}
-                    />
+                    <Header />
                 </header>
```

Comment:
esse header vai ficar vazio 
se for 
pode chamar so assim 

<header /> 

nao precisa abrir e fechar